### DISCUSSION TITLE
Remove or make optional the Permanent Password Restrictions

### DISCUSSION THREAD

**Discussion by nikkdrive18** [rustdesk/rustdesk#7869] [General]
Permanent Password format should be a suggestion instead of a restriction. Maybe give the option to allow insecure passwords with secure passwords option as default.

**Comment by themightyoarfish** [comment]
I find this very annoying as well. Not all applications require the same level of security. It's fine to nag the user about this, it's not fine to remove control from them.

**Comment by rainylibrary** [comment]
NIST, as of 2017, does not recommend complexity, but rather length. I would much prefer a "this password is short or weak, are you sure you want to use it" over an arbitrary set of requirements.

[NIST Special Publication 800-63B Digital Identity Guidelines, Appendix A](https://pages.nist.gov/800-63-3/sp800-63b.html#appA)
